eng/jacoco-test-coverage/pom.xml

```diff
@@ -67,20 +67,30 @@
       <version>1.6.0-beta.1</version> <!-- {x-version-update;com.azure:azure-core-http-netty;current} -->
     </dependency>
     <dependency>
-        <groupId>com.azure</groupId>
-        <artifactId>azure-core-http-okhttp</artifactId>
-        <version>1.3.0-beta.1</version> <!-- {x-version-update;com.azure:azure-core-http-okhttp;current} -->
-    </dependency>
-<!--    <dependency>-->
-<!--      <groupId>com.azure</groupId>-->
-<!--      <artifactId>azure-core-management</artifactId>-->
-<!--      <version>1.0.0-beta.8</version> &lt;!&ndash; {x-version-update;com.azure:azure-core-management;current} &ndash;&gt;-->
-<!--    </dependency>-->
+      <groupId>com.azure</groupId>
+      <artifactId>azure-core-http-okhttp</artifactId>
+      <version>1.3.0-beta.1</version> <!-- {x-version-update;com.azure:azure-core-http-okhttp;current} -->
+    </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-core-serializer-avro-apache</artifactId>
+      <version>1.0.0-beta.1</version> <!-- {x-version-update;com.azure:azure-core-serializer-avro-apache;current} -->
+    </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-core-serializer-avro-jackson</artifactId>
+      <version>1.0.0-beta.1</version> <!-- {x-version-update;com.azure:azure-core-serializer-avro-jackson;current} -->
+    </dependency>
```

Comment:
Are we not including JSON serializers?

Reply:
Splitting into two PRs, one for JSON and one for Avro.